index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```

Comment:
# Clean and tide code
Your code looks clean and you used semantic tags and, completed the changes requirement.